templates/include/brokered-space-infra.jsonnet

```diff
@@ -176,6 +176,7 @@ local hawkularBrokerConfig = import "hawkular-broker-config.jsonnet";
           }
         },
         "spec": {
+          "serviceAccount": "enmasse-service-account",
```

Comment:
This service account is not available in the address space namespace in a multitenant deployment so this must be created by the address controller. I’ll have a look.